tests/integration/test_dcim.py

```diff
@@ -17,115 +17,137 @@ class TestSimpleServerRackingAndConnecting:
     """Verify we can create, rack, and connect a server."""
 
     @pytest.fixture
-    def site(self, nb_client):
-        """Verify we can create a site."""
-        site_name = "MSP"
-        site = nb_client.dcim.sites.create(name=site_name, slug=site_name.lower().replace(" ", "_"), status="active")
-        assert site
+    def location(self, nb_client):
+        """Verify we can create a location."""
+        location_types = ["dcim.device", "dcim.rack", "dcim.rackgroup"]
+        site_type = nb_client.dcim.location_types.create(name="Site", content_types=location_types, nestable=True)
+        location_name = "MSP"
+        location = nb_client.dcim.locations.create(
+            name=location_name,
+            slug=location_name.lower().replace(" ", "_"),
+            status={"name": "Active"},
+            location_type=site_type.id,
+            depth=3,
```

Comment:
`depth` shouldn't have any effect on create calls, FWIW.

Reply:
Yes, I was testing that and I forgot to remove.